### DISCUSSION TITLE
Global Styles not applying to ChakraProvider

### DISCUSSION THREAD

**Discussion by dazp94** [chakra-ui/chakra-ui#2437] [Q&A] [answered]
Hello there,

I was following [global-styles documentation](https://next.chakra-ui.com/docs/features/global-styles) on a NextJS application to do some styling overriding but had no success whatsoever.

### What I did
In an attempt to avoid using a stylesheet for `body` styling, I had hoped that by creating a `config` object that would then be used by `extendTheme` to be passed as a `theme` prop to `ChakraProvided`.

<img width="452" alt="Screen Shot 2020-11-08 at 7 55 06 PM" src="https://user-images.githubusercontent.com/17577962/98482897-56ff9e00-21fc-11eb-875a-942dddd97b6d.png">

### What I expected
From the above implementation I would expect `body` to be styled as the below snippet.

<img width="504" alt="Screen Shot 2020-11-08 at 7 56 00 PM" src="https://user-images.githubusercontent.com/17577962/98482936-857d7900-21fc-11eb-9c0a-0e43736c7b43.png">

### What I got
As you can see, property `fontSize` was not computed as I expected

<img width="505" alt="Screen Shot 2020-11-08 at 7 56 22 PM" src="https://user-images.githubusercontent.com/17577962/98482938-88786980-21fc-11eb-958b-3b90637dc313.png">

Am I doing something wrong? 

Thank you for this amazing library!

**Comment by chasinhues** [comment]
It looks like the font size you've specified (`15`) might be what is causing the issue. 

You might be trying to use a unit-less value because it's possible for sizes (Chakra defines a set of spacing options for us to use - [https://next.chakra-ui.com/docs/theming/theme#spacing](https://next.chakra-ui.com/docs/theming/theme#spacing) ), but I don't think this will work as intended for font sizes (and more importantly, 15 isn't a space defined by chakra).

Check out the typography styles chakra offers by default ([https://next.chakra-ui.com/docs/theming/theme#typography](https://next.chakra-ui.com/docs/theming/theme#typography))
```
  fontSizes: {
    xs: "12px",
    sm: "14px",
    md: "16px",
    lg: "18px",
    xl: "20px",
    "2xl": "24px",
    "3xl": "28px",
    "4xl": "36px",
    "5xl": "48px",
    "6xl": "64px",
  },
```

Options that should work:
- `fontSize: "md"` (set the font size to the `md` value (`16px`)
- `fontSize: "15px"`

Alternatively, you can [customise the theme](https://next.chakra-ui.com/docs/theming/customize-theme) (override or add new font sizes) if you really want to add a 15px font size.

**Reply by dazp94** [reply]
Thank you for your reply @ChasinHues.

Actually fontSize is just an example for the sake of my problem (my styling is way bigger), since I can't apply any property to my styling.

However, using my example (`global` > `body` > `fontSize`) and applying both your suggestions, I still can't apply any change to `body`

I'd expect `ChakraProvider` would behave as a wrapper of `ThemeProvider` and `CSSReset`, but I can't apply any theming (besides dark mode, which is working).

**Comment by dazp94** [comment, marked as answer]
I ended up solving this.

The problem was I was passing only `config` as a theme, when I should be doing what I did in the screenshot below.

![Screen Shot 2020-11-10 at 9 54 03 AM](https://user-images.githubusercontent.com/17577962/98658375-e01cef00-233a-11eb-8039-b7c29a0a9fec.png)


Thank you for your help though!
